### PR TITLE
Fix static scan issue: dead code and unused value

### PR DESCRIPTION
**Pull request by jiaxuan-guo**
OAM-124684

### CODE REVIEW

File: drivers/dw3/dw3.c [hunk 1]
```diff
@@ -647,6 +647,9 @@ UsbdSetupHdlr (
 	                DEBUG ((DEBUG_INFO, "USB_REQ_SET_ADDRESS\n"));
 	                if (CtrlRequest->RequestType == USB_RT_TX_DIR_H_TO_D) {
 	                        Status = UsbdSetAddress ((UINT8)CtrlRequest->Value);
+							if (EFI_ERROR (Status)) {
+	              				DEBUG ((DEBUG_INFO, "UsbdSetAddress: EFI_DEVICE_ERROR\n"));
+	        				}
```
Comment (GangSecurity) [inline drivers/dw3/dw3.c]: no return ?

File: drivers/dw3/dw3.c [hunk 2]
```diff
@@ -590,6 +590,9 @@ UsbdSetupHdlr (
 	                                        CtrlRequest->Length,
 	                                        &(mCtrlIoReq.IoInfo.Length)
 	                                        );
+									if (EFI_ERROR (Status)) {
+										DEBUG ((DEBUG_INFO, "UsbdGetStringDesc() - EFI_DEVICE_ERROR"));
```
Comment (GangSecurity) [inline drivers/dw3/dw3.c]: return?

File: drivers/dw3/dw3.c (hunk 1 above)
Comment (jiaxuan-guo) [reply drivers/dw3/dw3.c]: Prefer to keep the previous code flow since this might just be a warning.

File: drivers/dw3/dw3.c (hunk 2 above)
Comment (jiaxuan-guo) [reply drivers/dw3/dw3.c]: same

**Review by GangSecurity — APPROVED** [review]

**Review by kwang13 — APPROVED** [review]

**Pull request merged**